tests/unit/options.js

```diff
@@ -1803,3 +1803,36 @@ exports.elision = function (test) {
 
   test.done();
 };
+
+// Option `trailingcomma` requires a comma after each element in an array or
+// object literal.
+exports.trailingcomma = function (test) {
+  var code = [
+    "var a = [];",
+    "var b = [1];",
+    "var c = [1,];",
+    "var d = [1,2];",
+    "var e = [1,2,];",
+    "var f = {};",
+    "var g = {a: 1};",
+    "var h = {a: 1,};",
+    "var i = {a: 1, b: 2};",
+    "var j = {a: 1, b: 2,};"
```

Comment:
Before you point out the irony in me *not* using a trailing comma here, I was simply following your contribution guidelines and keeping consistent with the dominant style in this file.